src/Uno.UWP/Devices/Midi/MidiInPort.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using Uno.Devices.Enumeration.Internal;
+using Windows.Foundation;
+
+namespace Windows.Devices.Midi
+{
+	/// <summary>
+	/// Represents a port used to receive MIDI messages from a MIDI device.
+	/// </summary>
+	public sealed partial class MidiInPort : IDisposable
+	{
+		private readonly static string MidiInAqsFilter =
+			"System.Devices.InterfaceClassGuid:=\"{" + DeviceClassGuids.MidiIn + "}\" AND System.Devices.InterfaceEnabled:=System.StructuredQueryType.Boolean#True";
+
+		/// <summary>
+		/// Gets the id of the device that was used to initialize the MidiInPort.
+		/// </summary>
+		public string DeviceId { get; private set; }
+
+		/// <summary>
+		/// Gets the id of the device that was used to initialize the MidiInPort.
+		/// </summary>
+		public event TypedEventHandler<MidiInPort, MidiMessageReceivedEventArgs> MessageReceived
+		{
+			add
+			{
+			}
+			remove
+			{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)